libraries/Microsoft.Bot.PublishValidation/Helpers/NotificationMessage.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Bot.PublishValidation
+{
+    public enum NotificationMessageTypes
+    {
+        Warning = 0,
+        Error = 1
+    }
+
+    public class NotificationMessage
+    {
+        public string message { get; set; }
+        public int type { get; set; }
```

Comment:
There's an Enum above. Why have this be an int?

Reply:
We already updated this class to use the Enum.